pyomo/gdp/plugins/cuttingplane.py

```diff
@@ -28,7 +28,7 @@
 )
 from pyomo.opt import SolverFactory
 
-from pyomo.gdp import Disjunct, Disjunction
+from pyomo.gdp import Disjunct, Disjunction, GDP_Error
```

Comment:
You're importing `GDP_Error` but never using it?

Reply:
GDP_Error is raised below.  This is fixing a bug/test failure from the 5.4.3 release.